Libplanet.Benchmarks/ProposeBlock.cs

```diff
@@ -29,7 +30,9 @@ public ProposeBlock()
         [Benchmark]
         public Block<DumbAction> ProposeBlockEmpty()
         {
-            return _blockChain.ProposeBlock(_privateKey);
+            return _blockChain.ProposeBlock(
+                _privateKey,
+                lastCommit: TestUtils.CreateBlockCommit(_blockChain.Tip));
```

Comment:
A bit confusing. Do we have to include block commit generation time to benchmark for `ProposeBlock`? I think it can be differed by the purpose of the benchmark.
May be, using `lastCommit` fixture and making another benchmark for generation of `BlockCommit` would be better for separating IMHO...

But seeing below benchmark, it checks both `ProposeBlock` and `Append`, so may be not.

Though, still confusing if it would be meaningful for including flexible time for `CreateBlockCommit`

Reply:
IMHO,
- block commit generation for various number of validators
- proposing block
- appending
would be better for future, if we consider not only benchmark for pow.